cypress/tsconfig.json

```
@@ -0,0 +1,7 @@
+{
+  "extends": "../tsconfig.json",
+  "include": ["*/*.ts"],
```

nit: would it work like this too?
```suggestion
  "include": ["."],
```

---

Working correctly with: `["."]`: [action run](https://github.com/valora-inc/dapp-list/actions/runs/4926526585/jobs/8802202182)!